src/data/mock/systemIntake.ts

```diff
@@ -1,4 +1,4 @@
-import { SystemIntakeWithReviewRequested } from 'gql/gen/types/SystemIntakeWithReviewRequested';
+import { SystemIntakeWithReviewRequestedFragment } from 'gql/gen/graphql';
```

Comment:
Caught this reference to the old folder/types, and it was a clean & easy update!